lib/scripts.js

```diff
@@ -82,12 +83,21 @@ exports.paths = {
  */
 exports.run = (script, callback) => {
   childProcess.execFile(script, (error, stdout, stderr) => {
+
+    // Platform scripts might output warnings to `stderr`
+    // that we don't want to take into consideration
+    // if the script exits with an exit code 0.
+    //
+    // If the scripts exits with a non-zero code, `.execFile()`
+    // yields an error whose message includes the `stderr`
+    // output, so we don't need to do anything ourselves.
+
     if (error) {
       return callback(error);
     }
 
     if (stderr.trim().length) {
-      return callback(new Error(stderr));
+      debug('stderr: %s', stderr);
```

Comment:
Yeah, this might require a lot of work. `debug()` is simply a wrapper around `console.error`, which we could stub, however `debug()` checks if the DEBUG environment variable is set when this file is first interpreted (therefore we can't modify it at runtime), so I guess testing this would be overkill.